app/src/main/res/values/strings.xml

```diff
@@ -206,6 +206,11 @@
   <string name="patientsummary_bangladesh_national_id">ID</string>
 
   <string name="patientsummary_contact_doctor">Doctor</string>
+  <string name="patientsummary_contact_doctor_diagnosis" translatable="false">*Diagnosis*</string>
+  <string name="patientsummary_contact_doctor_diagnosis_hypertension" translatable="false">Hypertension: </string>
+  <string name="patientsummary_contact_doctor_diagnosis_diabetes" translatable="false">Diabetes: </string>
```

Comment:
iirc, the Android build tools will remove blank spaces at the end of a message. Can we verify this? Might want to use the escape character instead.

Reply:
I think I can remove those spaces since I am adding a new line when using `StringBuilder`, so there is no point having those spaces will remove those.

Reply:
I am adding the space when building the message, so I can remove the spaces from the strings. Since we are planning on having hardcoded strings, either way, it makes sense I guess.